docs/import-roster-from-lms.md

```diff
@@ -0,0 +1,78 @@
+## Importing a course roster from your Learning Management System
+
+GitHub Classroom is able to automatically import your course roster from your institution's Learning Management
+System, potentially saving you time and headaches when provisioning a new GitHub Classroom course!
+
+### Prerequisites
+
+This guide assumes that you've successfully [connected GitHub Classroom to you Learning Management System](/help/connect-to-lms). If you haven't, please follow those setup instructions before proceeding to the next steps.
+
+Importing a course roster from your Learning Management System will only work if you have you have enabled GitHub Classroom to receive course roster information per the privacy settings within your Learning Management System.
+
+### Creating a new course roster
+
+To import a new course roster from your Learning Management, select the classroom you would like to import into. Then proceed through the following steps:
+
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/roster-import/import-roster.gif" class="border" style="width: 75%;">
+   </div>
+
+1. Navigate to classroom settings within GitHub Classroom
+
+2. Select "Roster Management" from the sidebar
```

Comment:
(for this whole section)

You can use formatting to make commands like this more reliable for skimming. Use _italics_ to show incidental UI text ("The _Foo settings_ page appears."); use **bold** text to indicate things you click or do other actions on ("Select **Roster management**.").

If you haven't already checked, make sure the UI text you write down is the same as appears in the UI, capitalization and all.